support-frontend/assets/pages/supporter-plus-landing/setup/setUpRedux.ts

```diff
@@ -179,28 +182,29 @@ function selectInitialContributionTypeAndPaymentMethod(
 	return contributionType;
 }
 
-function getStoredEmail(dispatch: ContributionsDispatch): void {
-	const sessionStorageEmail = storage.getSession('gu.email');
-	if (sessionStorageEmail) {
-		dispatch(setEmail(sessionStorageEmail));
-	}
-}
-
 export function setUpRedux(store: ContributionsStore): void {
 	const dispatch = store.dispatch;
 	const state = store.getState();
 	// TODO - move these settings out of the redux store, as they only change once, upon initialisation
 	const contributionTypes = getContributionTypes(state);
 	dispatch(setContributionTypes(contributionTypes));
-	getStoredEmail(dispatch);
+
+	setUpUserState(dispatch);
```

Comment:
My understanding is this where the thunk (asynchronously) initialises User.supporterStatus from the user-attributes json file? From a recurring contributor.

Reply:
`getRecurringContributorStatus` is the thunk, yes, it talks to MDAPI to get the user's current product status.